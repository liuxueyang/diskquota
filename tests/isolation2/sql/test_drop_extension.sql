SELECT diskquota.pause();
SELECT diskquota.wait_for_worker_new_epoch();
DROP EXTENSION diskquota;
