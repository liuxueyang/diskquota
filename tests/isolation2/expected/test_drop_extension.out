SELECT diskquota.pause();
 pause 
-------
       
(1 row)
SELECT diskquota.wait_for_worker_new_epoch();
 wait_for_worker_new_epoch 
---------------------------
 t                         
(1 row)
DROP EXTENSION diskquota;
DROP
